--- origsrc/passepartout-0.7.0/src/pptout/Makefile.am	2007-07-26 06:31:05.000000000 -0500
+++ src/passepartout-0.7.0/src/pptout/Makefile.am	2007-07-27 01:35:00.234375000 -0500
@@ -37,14 +37,13 @@
 
 passepartout_CXXFLAGS += -I$(top_srcdir)/src -D'XMLPATH=$(xmldir)'
 
-passepartout_LDADD = $(XMLPP_LIBS) $(GTKMM_LIBS) $(GNOMECANVAS_LIBS) \
-		$(GTHREAD_LIBS) $(GNOME_LIBS) $(GNOMEVFS_LIBS)
-
 pptpartlibs= document/libdocument.a ../xml2ps/libxml2ps.a	\
 	../fonts/libfonts.a ../ps/libps.a			\
 	../util/libpptutil.a widget/libwidget.a
 
-passepartout_LDADD += $(pptpartlibs)
+passepartout_LDADD = $(pptpartlibs) $(XMLPP_LIBS) $(GTKMM_LIBS) \
+		$(GNOMECANVAS_LIBS) $(GTHREAD_LIBS) $(GNOME_LIBS) $(GNOMEVFS_LIBS)
+
 
 # Extra dependencies for builddate, so it is rebuilt every time anything else
 # is rebuilt
--- origsrc/passepartout-0.7.0/src/ps/Makefile.am	2007-06-13 06:48:09.000000000 -0500
+++ src/passepartout-0.7.0/src/ps/Makefile.am	2007-07-27 01:35:00.234375000 -0500
@@ -15,8 +15,8 @@
 libps_a_CXXFLAGS += -I$(top_srcdir)/src
 
 examinepdf_CXXFLAGS = $(libps_a_CXXFLAGS)
-examinepdf_LDADD = $(GTKMM_LIBS) \
-	-L. -lps -L../fonts -lfonts -lps -L../util -lpptutil
+examinepdf_LDADD = libps.a ../fonts/libfonts.a libps.a ../util/libpptutil.a \
+		$(GTKMM_LIBS)
 examinepdf_DEPENDENCIES = libps.a
 
 #libps_a_LIBADD = $(GTKMM_LIBS)
@@ -27,5 +27,5 @@
 runtest_CXXFLAGS = -I$(top_srcdir)/src
 runtest_CXXFLAGS += $(GTKMM_CFLAGS)
 runtest_DEPENDENCIES = ../util/libpptutil.a $(noinst_LIBRARIES)
-runtest_LDADD = -L../testbed -lppttest -L. -lps -L../util -lpptutil \
+runtest_LDADD = ../testbed/libppttest.a libps.a ../util/libpptutil.a \
 		$(GTKMM_LIBS)
--- origsrc/passepartout-0.7.0/src/util/Makefile.am	2007-06-14 05:12:42.000000000 -0500
+++ src/passepartout-0.7.0/src/util/Makefile.am	2007-07-27 01:35:00.250000000 -0500
@@ -20,4 +20,4 @@
 	test_boundary.cc test_procio.cc test_cmdline.cc
 runtest_CXXFLAGS = -I$(top_srcdir)/src
 runtest_DEPENDENCIES = $(noinst_LIBRARIES)
-runtest_LDADD = -L../testbed -lppttest -L. -lpptutil $(GTKMM_LIBS) $(GNOMECANVAS_LIBS)
+runtest_LDADD = ../testbed/libppttest.a libpptutil.a $(GTKMM_LIBS) $(GNOMECANVAS_LIBS)
--- origsrc/passepartout-0.7.0/src/xml2ps/Makefile.am	2007-06-22 11:07:26.000000000 -0500
+++ src/passepartout-0.7.0/src/xml2ps/Makefile.am	2007-07-27 01:35:00.265625000 -0500
@@ -20,8 +20,9 @@
 
 xml2ps_CXXFLAGS = $(libxml2ps_a_CXXFLAGS)
 
-xml2ps_LDADD = $(XMLPP_LIBS) $(GTKMM_LIBS) $(GNOMECANVAS_LIBS)\
-	libxml2ps.a ../fonts/libfonts.a ../ps/libps.a ../util/libpptutil.a 
+xml2ps_LDADD = \
+	libxml2ps.a ../fonts/libfonts.a ../ps/libps.a ../util/libpptutil.a \
+	$(XMLPP_LIBS) $(GTKMM_LIBS) $(GNOMECANVAS_LIBS)
 
 #install DTDs and XSLTs in $(datadir)/xml/passepartout/ (xmldir is defined in configure.ac)
 dist_xml_DATA = xml2ps.dtd xhtml.xslt docbook.xslt
